src/components/profile/UserProfilePage/style.module.scss

```diff
@@ -0,0 +1,215 @@
+@use 'src/styles/vars.scss' as vars;
+
+.profilePage {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  margin: auto;
+  max-width: vars.$breakpoint-md;
+  width: 100%;
+
+  .cardWrapper {
+    padding-top: 3rem;
+    position: relative;
+
+    .banner {
+      background-color: vars.$pink-2;
+      border-radius: 0.5rem;
+      height: 90%;
+      position: absolute;
+      top: 0;
+      width: 100%;
+    }
+
+    .profileCard {
+      background-color: var(--theme-background);
+      border-radius: 0.25rem;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
+      display: flex;
+      gap: 1rem;
+      margin: 0 2rem;
+      padding: 1rem 2rem;
+      position: relative;
+
+      @media (max-width: vars.$breakpoint-md) {
+        align-items: center;
+        flex-direction: column;
+      }
+
+      .profilePic {
+        border-radius: 4rem;
+        flex-shrink: 0;
+        height: 7rem;
+        overflow: hidden;
+        position: relative;
+        width: 7rem;
+      }
+
+      .cardName {
+        align-items: flex-start;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        @media (max-width: vars.$breakpoint-md) {
+          align-items: center;
+          flex-direction: column;
+          
+          h1 {
+            text-align: center;
+          }
```

Comment:
you should also center the handle

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/0d861918-c776-4b00-bc7a-61beb20f1b60)

Reply:
Implemented in latest commit https://github.com/acmucsd/membership-portal-ui-v2/commit/8a0198fd280f38cb33d2fb9007c79be99804e06c 👍